### ISSUE TITLE
Tokenisation error

### ISSUE THREAD

**Issue by bguil** [UniversalDependencies/UD_Japanese-GSD#15]
There is only one exception to the following Grew pattern which search for an `obj` argument without the "を" particle (see it [online](http://match.grew.fr/?corpus=UD_Japanese-GSD@2.4&custom=5db606dd252d8)):

```
pattern { GOV -[obj]-> DEP; GOV [upos=VERB]; }
without { DEP -[case]-> S; S [phon="を"] }
```

I guess that the token "、を" should be split.

**Comment by kanayamah** [comment]
Thank you.   In this case the original sentences itself is broken: a word was somehow dropped.
It should be ``即位後まもなく後趙の麻秋が来攻したが、**謝艾**を遣わしてこれを大いに破った。``

I will drop the sentence or modify the original text in the next release.

**Comment by kanayamah** [comment]
This problem will be solved in the next release (v2.6).
